api/src/main/java/org/eclipse/microprofile/metrics/MetricID.java

```diff
@@ -71,6 +73,8 @@
 
     public static final String GLOBAL_TAGS_VARIABLE = "mp.metrics.tags";
 
+    public static final String GLOBAL_TAGS_ENV_VARIABLE = "MP_METRICS_TAGS";
```

Comment:
This already works without your PR, you can pass an env property `MP_METRICS_TAGS`,  and because of MP Config's mapping rules (https://github.com/eclipse/microprofile-config/blob/1.3/spec/src/main/asciidoc/configsources.asciidoc#default-configsources) it will be accessible at runtime under the `mp.metrics.tags` name. It's also mentioned in the MP Metrics spec: https://github.com/eclipse/microprofile-metrics/blob/2.2/spec/src/main/asciidoc/architecture.adoc#tags

This makes me realize that the javadoc is a bit wrong, actually you are changing that into the correct form, because this should already work (if the used MP Config implementation is correct). 

So how about you change just the javadoc? And leave the rest unchanged?

Reply:
Apologies, I overlooked the Config section and bit confused with JavaDoc.